7_react/react1/week3/my-app/src/CreateTask/index.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react'
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+export default function CreateTask({ addNewTask }) {
+    const [description, setDescription] = useState("");
+    const [deadline, setDeadline] = useState(new Date());
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        if (!description) {
+            return window.alert('Please add a task description')
+        }
+
+        const date = new Date()
+
+        deadline.setHours(0, 0, 0, 0)
+
+        date.setHours(0, 0, 0, 0)
+
+        if (deadline < date) {
+            return window.alert('Please select a deadline in present or future')
```

Comment:
Since handleSubmit does not expect a return value, you can rewrite this into:
```
alert('...');
return;
```

Here the purpose of the return statement will be simply to stop executing the rest of the function, which is correct.